src/interfaces/tag-type-response.ts

```diff
@@ -0,0 +1,25 @@
+import { PillCategory } from '../types/pill-category';
+
+interface GenericTagTypeResponse {
+    name: string;
+    calculatedSlug: string;
+}
+
+export interface ExpertiseLevelTag extends GenericTagTypeResponse {}
+
+export interface ContentTypeTag {
+    contentType: PillCategory;
```

Comment:
Can we just call this field `name` rather than `contentType` to keep it in line with the structure of the generic tag interface?

Reply:
the way I created strapi , I cudn't manipulate that field on graphql, thats why this had to be an exception